mat/qr.go

```diff
@@ -118,23 +123,27 @@ func (qr *QR) RTo(dst *Dense) *Dense {
 	for i := r; i < c; i++ {
 		zero(dst.mat.Data[i*dst.mat.Stride : i*dst.mat.Stride+c])
 	}
-
-	return dst
 }
 
-// QTo extracts the m×m orthonormal matrix Q from a QR decomposition.
-// If dst is nil, a new matrix is allocated. The resulting Q matrix is returned.
-// QTo will panic if the receiver does not contain a factorization.
-func (qr *QR) QTo(dst *Dense) *Dense {
+// QTo extracts the r×r orthonormal matrix Q from a QR decomposition.
+//
+// If dst is empty, QTo will resize dst to be r×r. If dst is non-empty,
+// QTo will panic if dst is not r×r. QTo will also panic if the receiver
```

Comment:
"When dst is non-empty, QTo will panic if dst is not r×r."

Reply:
Done